pkg/models/image.go

```diff
@@ -16,7 +16,7 @@ type ImageReader interface {
 	// CountByStudioID(studioID int) (int, error)
 	// CountByTagID(tagID int) (int, error)
 	All() ([]*Image, error)
-	Query(imageFilter *ImageFilterType, findFilter *FindFilterType) ([]*Image, int, error)
+	Query(imageFilter *ImageFilterType, findFilter *FindFilterType) ([]*Image, int, float64, int, error)
```

Comment:
I think that we're going to need to refactor how we do this interface. Returning this many values is ugly. 

I'm guessing we probably want a `ImageQueryResult` struct that returns the image objects and other stuff. I also think that we should probably accept a query options object so that we don't always have to use potentially slower queries.

It also might be a good opportunity to return ids only and have the object-resolving code pushed out to the caller.